_includes/tutorials/connect-sink-timestamp/ksql/code/tutorial-steps/dev/create-stream.sql

```diff
@@ -0,0 +1,2 @@
+CREATE STREAM TEMPERATURE_READINGS_RAW (eventTime BIGINT, temperature int)
```

Comment:
Presumably it's for demo purposes, but FYC explain why eventTime is set to `BIGINT` at stream creation

Reply:
I addressed this in one of my latest commits. After creating the stream I mention:
"The schema for our example stream specifies `eventTime` as a `BIGINT`. It is common for applications to send data in Unix time, thus why `eventTime` is specified as a `BIGINT`."